### PR TITLE
Depoly real env found issues fix

### PR DESCRIPTION
**Pull request by bzhaoopenstack**
This PR fix list:
1. disable zuul debug level
2. Install openstackclient on all nodes for fitting checker requirements
3. Fix some nodes lacks setuptools before using pip install
4. Fix zookeeper local configuration to bind local interface ip.
5. Fix "create modsecurity config file" task in apache.yaml if we run 2nd time labkeeper, it will fail
6. Fix zuul-web exception to correct the binding IP in zuul.conf
7. Fix zuul-executor exception as lacking ansible 2.5 2.6 2.7
8. Fix zuul scheduler exception for correct zuul keys dir from 755 to 700
9. Fix nodepool build image failed as openstack-dev and openstack-infra

Related-Bug: theopenlab/openlab#218

### CODE REVIEW

File: playbooks/prerequisites.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@
       package:
         name: "{{ item }}"
       with_items:
+        - python-setuptools
```
Comment (wangxiyuan) [inline playbooks/prerequisites.yaml:30]: python3-setuptools should be added as well.

**Review by wangxiyuan — CHANGES_REQUESTED** [review]

**Review by huangtianhua — APPROVED** [review]

**Comment by wangxiyuan** [comment]
Let's wait a moment in order to fix all related bug in one PR.

File: playbooks/prerequisites.yaml [hunk 2]
```diff
@@ -27,6 +27,8 @@
...
+        - python3-setuptools
```
Comment (Yikun) [inline playbooks/prerequisites.yaml:31]: and `openstack` commands?

File: openlabcmd/requirements.txt [hunk 3]
```diff
@@ -1,6 +1,7 @@
 configparser
 enum;python_version=='2.7'
```
Comment (wangxiyuan) [inline openlabcmd/requirements.txt:2]: not related to this patch:  enum can be removed, since openlabcmd is only for py3 now.

**Comment by Yikun** [comment]
recheck

**Comment by wangxiyuan** [comment]
recheck

**Comment by bzhaoopenstack** [comment]
recheck

**Review by wangxiyuan — APPROVED** [review]

**Pull request merged**